### PR TITLE
Briefly explain the contents of the repo in readme

### PR DESCRIPTION
**Pull request by aofarrel**
This will be useful for not just users, but ourselves, especially as the repo grows.

### CODE REVIEW

**Comment by aofarrel** [comment]
I mixed up the blood pressure and 1000 genomes notebooks *again*

I'll swap that part out in a few, whoops

**Comment by aofarrel** [comment]
oh that works too

File: README.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ Example Jupyter notebook source files for the BioData Catalyst consortium. These
 Jupyter notebook `.ipynb` files are generated from source files with the
 [herzog](https://github.com/xbrianh/herzog) library. 
 
+## Notebooks Represented
+* BYOD -- Tutorial for bringing your own data to the Terra platform
+* GWAS Blood Pressure -- Two part series teaching how to use Gen3 data to preform a GWAS; [mirror of template workspace](https://app.gitbook.com/@bdcatalyst/s/biodata-catalyst-tutorials/tutorials/terra-tutorials/bdcatalyst-gwas-terra-gen3-dockstore)
```
Comment (xbrianh) [inline README.md:10]: s/preform/perform

**Review by xbrianh — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (aofarrel) [reply README.md:10]: heck

**Review by juklucas — APPROVED** [review]
lgtm (after incorporating Brian's spelling fix)

**Pull request merged**